src/Makefile.global.in

```diff
@@ -598,6 +598,13 @@ TAS         = @TAS@
 
 ##########################################################################
 #
+# Link to ZMQ
+
+LIBS := -lzmq $(LIBS)
```

Comment:
I guess for a development build using it as a regular library is fine. But for packages I don't see any reason not to build it as a static library with `g++`.